src/cdk/private/visually-hidden/visually-hidden.scss

```diff
@@ -0,0 +1,34 @@
+/// This class can be applied to an element to make that element
+/// visually hidden while remaining available to assistive technology.
+.cdk-visually-hidden {
```

Comment:
Could we reuse the `a11y-visually-hidden` mixin here instead of copying the styles?